README.md

```diff
@@ -1,4 +1,4 @@
-uvcdat
+uv-cdat
```

Comment:
> Shouldn't this be UV-CDAT?

this is not from my commit but rather came from Charle's merge. I am suspecting this is what we have in the master.